klab.engine/src/main/java/org/integratedmodelling/klab/components/geospace/processing/osm/OSMSubjectInstantiator.java

```diff
@@ -335,6 +345,9 @@ private String buildQuery() throws KlabValidationException {
 		if (this.notmatching != null) {
 			query.filterNotMatch(this.notmatching.toArray());
 		}
+        if (!this.conditions.isEmpty()) {

```

Comment:
Why are we using `isEmpty()` here and `isBlank()` in the other class? Is that intended to be like that? 🤔 
https://www.baeldung.com/java-string-isempty-vs-isblank#comparing-isblank-vs-isempty